### DISCUSSION TITLE
如何处理空指令

### DISCUSSION THREAD

**Discussion by lztlzt-1** [nonebot/discussions#363] [问答]
假设我的机器人NICKNAME设置了“bot1”，我想在任何人发送bot1时回答“我在”，我该如何做
尝试过@on_command('callme',only_to_me=True,aliases=('说话',None))，@on_command('callme',only_to_me=True,aliases=('说话',‘’))
但都无法正确处理

**Comment by yanyongyu** [comment]
nb2写法：`on_message(rule=to_me())`

**Reply by lztlzt-1** [reply]
我的nonebot是通过 https://docs.nonebot.dev/ 的介绍下载的，我在nonebot里import发现没有on_message
![Uploading image.png…]()

**Reply by yanyongyu** [reply]
nb1你需要用natual language来实现

**Reply by lztlzt-1** [reply]
谢谢，已经完成了，方法是通过`@on_natural_language(keywords={""},allow_empty_message=True)`传入处理函数
